crates/net/src/lib.rs

```diff
@@ -64,7 +66,13 @@ async fn discover_peers(udp_addr: SocketAddr, bootnodes: Vec<BootNode>) {
             Message::Neighbors(neighbors_msg) => {
                 let nodes = &neighbors_msg.nodes;
                 for node in nodes {
-                    table.insert_node(*node);
+                    let peer_data = PeerData {
+                        ip: node.ip,
+                        udp_port: node.udp_port,
+                        tcp_port: node.tcp_port,
+                        node_id: node.node_id,
+                    };
```

Comment:
Let's add a `From<Node> for PeerData` impl

Reply:
Done!